Source/Src_nd/Derive_PLM_nd.F90

```diff
@@ -1246,10 +957,10 @@ subroutine dcma (e,   e_lo, e_hi, nv, &
       OH = get_species_index('OH')
       RO2 = get_species_index('C12H25O2')
 
-      CALL  dermixfrac(e(:,:,:,1),  e_lo, e_hi, 1, &
-                       dat(:,:,:,rho:NUM_SPECIES+1), d_lo, d_hi, NUM_SPECIES+1, &
-                       lo, hi, domlo, domhi, delta, xlo, time, dt, bc, &
-                       level, grid_no)
+      !CALL  dermixfrac(e(:,:,:,1),  e_lo, e_hi, 1, &
```

Comment:
this routine is now broken here, right?

Reply:
it's not broken, just incomplete. But it's going to be removed very shortly -working on the CPP version right now. I could put back the fortran version but...is anyone going to use it anyways right now ?